originals/c.yaml

```diff
@@ -95,6 +95,26 @@
     theme:
     - Fantasy
 
+- name: Catacomb 3-D
+  external:
+    wikipedia: Catacomb 3-D
+  meta:
+    genre:
+    - Action
+    - FPS
+    theme:
+    - Fantasy
+    
+- name: Catacomb Adventure Series
```

Comment:
If this is the same game as Catacomb 3-D, we shouldn't add it here. Instead we can have it as an alternate name to Catacomb 3-D

Reply:
Not the same game.
The Adventure Series is the name of the trilogy ( Catacomb Abyss, Catacomb Armageddon, Catacomb Apocalypse ). The Adventure Series the sequel to the 3-D.